core/commands/pin.go

```diff
@@ -250,21 +250,11 @@ Defaults to "direct".
 				return nil, u.ErrCast()
 			}
 			out := new(bytes.Buffer)
-			if typeStr == "indirect" && count {
```

Comment:
i believe this needs to be brought back. (with likely a mark and sweep implementation)